BUILD.md

```diff
@@ -42,3 +42,17 @@ The progress of the build can be monitored with:
 Once built, the image will be visible in the Image Stream created by the same command:
 
         oc describe imagestream openshift-ansible
+
+## Build the System Container
```

Comment:
Certainly, not being very familiar with system containers myself yet (except for your talk at DevConf :smile: ), that seems great! Thanks.